src/shared/proto/tbots_robot_msg.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto2";
+
+import "shared/proto/robot_component_status.proto";
+
+message TbotsRobotMsg
```

Comment:
Should we add a test that builds this message with all the fields required for the old robots and verifies that it doesn't go over 100 bytes? I feel like we are getting pretty close if not already over the limit

Reply:
testing replying to a comment @garethellis0